apache/superset
# chore: Select component refactoring - SelectAsyncControl - Iteration 5

### SUMMARY
This PR replaces the react-select Select with the Antdesign Select in the SelectAsyncControl component.

### BEFORE/AFTER

https://user-images.githubusercontent.com/60598000/132246572-865ec0c7-3141-4295-a4dd-d00daea79ad6.mp4

https://user-images.githubusercontent.com/60598000/132246613-c726e613-e6a6-46fa-a608-fc321e1b669d.mp4


### TESTING INSTRUCTIONS
1. Visit Explore
2. Click on "Edit dataset"
3. Click on "Settings"
4. Change owners and observe the behavior of the Select

### ADDITIONAL INFORMATION
